docs/dev/HadoopWinutils.md

```diff
@@ -0,0 +1,11 @@
+## Hadoop Winutils
+
+Hadoop Winutils native libraries are required to run Drill on Windows. The last version presented in maven repository is 2.7.1 and doesn't updated anymore.
```

Comment:
presented in maven repository -> present in Maven repository
doesn't updated anymore -> is not updated

Reply:
Fixed.